src/tagging/components/tagModifier.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TagSelector from './tagSelector';
+import ValueSelector from './valueSelector';
+
+const TagModifier = (props) => {
```

Comment:
please extact the strings, so you can do 
```js
const TagModifier = ({ tags, selectedTagCategory, onTagCategroyChange... })
```
and then just use them without the props.<>

Reply:
Done